packages/harpguru-core/src/utils/ismatch-highordertuples/ismatch-highordertuples.ts

```diff
@@ -0,0 +1,12 @@
+// TODO: This could use a test file
```

Comment:
It could, but perhaps hold off until we've made a decision about the layoutfacts type.

Reply:
Removed in https://github.com/js-jslog/harpguru/pull/130.